content/blog/20240430_fluentd-v1.17.0-has-been-released.md

```diff
@@ -63,7 +63,142 @@ Please see [#4282](https://github.com/fluent/fluentd/pull/4282) for more informa
 
 ### Make sure parser returns hash
 
-TODO
+The record data in a Fluentd event must be a hash object.
+
+* [Life of a Fluentd event](https://docs.fluentd.org/quickstart/life-of-a-fluentd-event)
+
+However, in the previous versions, some parser plugins could return a non-hash object, such as an array.
+It could cause errors in subsequent processing.
+
+In this release, the following parser plugins have been fixed.
+
+* [parser_json](https://docs.fluentd.org/parser/json)
+* [parser_msgpack](https://docs.fluentd.org/parser/msgpack)
+
+The changes are as follows.
```

Comment:
To indicate enumerating items, we could use the last comma instead of the period.
```suggestion
The changes are as follows:
```